### PR TITLE
Added a list of 'model-types' (and other refinements)

### PR DESCRIPTION
**Pull request by gurcei**
Various refinements, but the main point of interest for the PR was the list of model-types I added to the 'cores-and-flashing' chapter.

I was only recently made aware of the release model having the larger QSPI flash (64MB?), so wanted to confirm I've understood this correctly.

It'd be good if @gardners can confirm I'm on the right wavelength with this.

### CODE REVIEW

File: configuring.tex [hunk 1]
```diff
@@ -92,47 +97,53 @@ \section{Preparing for First Use: Formatting SD Cards}
...
+\item {\tt C65THUMB.M65} (C65 thumbnail image used in freezer)
+\item {\tt MEGA65.ROM}   (128KB ROM file)
 \item {\tt MEGA65.D81 (default disk image, automatically mounted)}
 \end{itemize}
 
-Straight out-of-the-box, MEGA65 will only have one SD card installed, located inside behind the trap-door. This SD card contains the essential files to properly boot from.
-When an external microSD card is inserted, MEGA65 consider the external one as higher priority and will try to boot from it.
-That means that the microSD card needs to have the essential files on it, otherwise MEGA65 cannot boot properly and will fall back into loading the OpenROM which does not support all the MEGA65 features.
-In general, if MEGA65 cannot boot properly and fall back to OpenROM, it will be shown in the power-up screen, similar to this:
+Straight out-of-the-box, MEGA65 will only have one internal SD card installed, accessible via the trap-door under the case. This SD card contains all essential files needed to properly boot up.
```
Comment (Impakt) [inline configuring.tex:110]: not convinced out of the box needs hyphens?

... out of the box, _the_ MEGA65

File: configuring.tex [hunk 2]
```diff
@@ -92,47 +97,53 @@ \section{Preparing for First Use: Formatting SD Cards}
...
+If an external microSD card is also detected during boot up, the MEGA65 gives it higher priority and will try to boot from it.
```
Comment (Impakt) [inline configuring.tex:111]: from it instead

File: configuring.tex [hunk 3]
```diff
@@ -92,47 +97,53 @@ \section{Preparing for First Use: Formatting SD Cards}
...
 
 \section{Installing ROM and Other Support Files}
 \label{sec:installingrometc}
 
-The MEGA65 FDISK/FORMAT utility will install a version of the
-open-source OpenROM project's C64-compatible ROMs as part of the
-formatting process.
-The default ROM however is read from the file MEGA65.ROM, which contains
+The MEGA65 FDISK/FORMAT utility will add to your SD card a version of the
+open-source 'Open ROMs' project's C64-compatible ROM, and will name it MEGA65.ROM.
```
Comment (Impakt) [inline configuring.tex:122]: will add a copy of the Open ROMs project's C64-compatible ROM to your SD card?

File: configuring.tex [hunk 4]
```diff
@@ -92,47 +97,53 @@ \section{Preparing for First Use: Formatting SD Cards}
...
+
+For MEGA65 owners, we have replaced this file with the latest ROM from the 'Closed ROMs'
+project. It provides many improvements over original/incomplete C65 ROMs. It contains
```
Comment (Impakt) [inline configuring.tex:125]: over _the_ original C65 ROMs

File: configuring.tex [hunk 5]
```diff
@@ -92,47 +97,53 @@ \section{Preparing for First Use: Formatting SD Cards}
...
 SD card.  Make sure that they have the .ROM extension.  The default ROM
-should be called MEGA65.ROM.  These files
+should be called MEGA65.ROM. These files
 must be 128KB in size, and use the same internal format as ROMs
 intended for the C65.  This means that the C64-mode KERNAL must be
 placed at offset \$E000, a C65-mode BASIC at \$A000, and a suitable
 character set at \$D000.
 
+You can optionally name your alternate ROMs as 'MEGA65x.ROM', replacing 'x' with a number from 0 to 9.
+This allows you to quickly boot-up to your alternate ROMs by holding down a number from 0 to 9 prior
```
Comment (Impakt) [inline configuring.tex:141]: Up to you, but you might want to add the megakey tags here

File: configuring.tex [hunk 6]
```diff
@@ -92,47 +97,53 @@ \section{Preparing for First Use: Formatting SD Cards}
...
 character set at \$D000.
 
+You can optionally name your alternate ROMs as 'MEGA65x.ROM', replacing 'x' with a number from 0 to 9.
+This allows you to quickly boot-up to your alternate ROMs by holding down a number from 0 to 9 prior 
+to powering on your MEGA65.
+
 Other important support files include FREEZER.M65 and AUDIOMIX.M65, which
-allow you to use the MEGA65's integrated freezer. More details provided below.
+allow you to use the MEGA65's integrated freezer. More details provided in the 'Floppy Disks And D81 Images' 
+chapter on page \pageref{cha:freezer}.
```
Comment (Impakt) [inline configuring.tex:146]: More details _are_ provided

File: configuring.tex [hunk 7]
```diff
@@ -141,27 +152,29 @@ \subsection{ROM File}
 You may want to source your own separate C65 ROM via other means.
 There were many different versions created during the development of the Commodore 65,
 and the MEGA65 can use any of them.  However, they will not support the advanced
-features of the MEGA65 and are incomplete and buggy, because the development
-was stopped, before these C65 ROMs could be completed.
+features of the MEGA65 and are incomplete and buggy, as development on them ceased
```
Comment (Impakt) [inline configuring.tex:155]: features of the MEGA65, (added a comma)

File: configuring.tex [hunk 8]
```diff
@@ -141,27 +152,29 @@ \subsection{ROM File}
...
+due to Commodore abandoning the C65 project.
 
-\textbf{MEGA65 ROMs}
+\textbf{MEGA65 Closed ROMs}
 
-There are newer versions of the \textbf{MEGA65 Closed ROM} actively under development also. These ROMs improve upon the original C65 ROMs and make better use of the extra hardware capabilities that the MEGA65 has over the original C65 hardware. These ROMs are available via the filehost also, but only to owners of the MEGA65, who will need to log into the filehost with their credentials in order to download it. It can be located by visiting the "\textbf{Files}" tab and searching for "\textbf{kernel rom}":
+There are newer versions of the \textbf{MEGA65 Closed ROM} actively under development also. These ROMs improve upon the original C65 ROMs and make better use of the extra hardware capabilities that the MEGA65 has over the original C65 hardware. These ROMs are available via the filehost, but only to owners of the MEGA65, who will need to log into the filehost with their credentials in order to download it. It can be located by visiting the "\textbf{Files}" tab and searching for "\textbf{kernel rom}":
```
Comment (Impakt) [inline configuring.tex:160]: There are _also_, and remove the also at the end of the sentence.

File: configuring.tex [hunk 9]
```diff
@@ -141,27 +152,29 @@ \subsection{ROM File}
...
 
 \includegraphics[width=\linewidth]{images/latest_closed_rom.png}
 
 \textbf{MEGA65 ROM diff files}
 
-If you have sourced your own 911001.bin C65 ROM and would like to patch it to the latest MEGA65 ROM, we do provide patches, as the additional improvements we have made to the closed rom are open source. Those diff files are available here:
+If you have sourced your own 911001.bin C65 ROM and would like to patch it to the latest MEGA65 Closed ROM, 
+we do provide patches, as the additional improvements we have made to the closed rom are open source.
```
Comment (Impakt) [inline configuring.tex:167]: closed ROM

File: configuring.tex [hunk 10]
```diff
@@ -141,27 +152,29 @@ \subsection{ROM File}
...
 
 \textbf{MEGA65 ROM diff files}
 
-If you have sourced your own 911001.bin C65 ROM and would like to patch it to the latest MEGA65 ROM, we do provide patches, as the additional improvements we have made to the closed rom are open source. Those diff files are available here:
+If you have sourced your own 911001.bin C65 ROM and would like to patch it to the latest MEGA65 Closed ROM, 
+we do provide patches, as the additional improvements we have made to the closed rom are open source. 
+Those diff files are available here:
 
-\url{https://files.mega65.org?id=fd2c40b9-f337-41f7-8a81-0254b1e09fb5}
+\url{http://mega65.org/rom-diffs}
```
Comment (Impakt) [inline configuring.tex:170]: feels good.jpg!

File: cores-and-flashing.tex [hunk 11]
```diff
@@ -35,7 +35,36 @@ \section{What are cores, and why does it matter?}
 of these systems (once the cores are available), and back again.
 
 The remainder of this chapter describes how to select which core to run on the MEGA65, and
-how to store a core into one of the three (or for MEGA65R2, seven) slots in the flash memory storage.
+how to store a core into one of the seven slots in the flash memory storage.
+
+\subsection{Model types}
+
+The retail model you have purchased is referred to as the MEGA65R3b (revision 3b). Throughout the course of development of the MEGA65, there have been several other model variants used by developers, each with differing specifications and available core slots, so they will be listed here, just to raise awareness of them.
```
Comment (Impakt) [inline cores-and-flashing.tex:42]: 3A or 3B? note the capital also

**Review by Impakt — COMMENTED** [review]
@gurcei made some comments, up to you how whether you want to address them or not

File: configuring.tex (hunk 1 above)
Comment (gurcei) [reply configuring.tex:110]: Yep sounds good, will straighten this out.

File: configuring.tex (hunk 5 above)
Comment (gurcei) [reply configuring.tex:141]: Hehe, yeah, I was being lazy :D Ok, will add them in.

File: cores-and-flashing.tex (hunk 11 above)
Comment (gurcei) [reply cores-and-flashing.tex:42]: Yeah, this 3a/3b idea was just an initial stab in the dark until Paul and Deft filled me in with their preference. Okidoke, will update to their preference now.

File: configuring.tex [hunk 12]
```diff
@@ -3,6 +3,11 @@ \chapter{Configuring your MEGA65}
 
 \section{Preparing for First Use: Formatting SD Cards}
 
+\textbf{NOTE}: For your convenience, your MEGA65 comes with an SD card with all essential 
+files placed on it. So you may prefer to skip this section and jump to the on-boarding section
```
Comment (Impakt) [inline configuring.tex:7]: comma here between it and So?

File: configuring.tex (hunk 5 above)
Comment (Impakt) [reply configuring.tex]: not sure if boot-up needs a hyphen, again up to you

File: configuring.tex (hunk 12 above)
Comment (gurcei) [reply configuring.tex:7]: Hmm, I'm 50/50 on it, but ok, let's try it that way :)

**Review by Impakt — APPROVED** [review]

File: configuring.tex (hunk 5 above)
Comment (gurcei) [reply configuring.tex]: I refined my fix here a little to just use \megakey for the 2nd mention of the 0 to 9 range (and removed it from the first mention of the range), seemed to look nicer in the pdf that way.

File: configuring.tex (hunk 6 above)
Comment (gurcei) [reply configuring.tex:146]: Oops, missed this one, seems to have gotten hidden in the web interface, ok fixed this too.

**Pull request merged**